news/2021-11-08-performance-points-star-rating-updates.md

```diff
@@ -0,0 +1,132 @@
+---
+layout: post
+title: "Performance Points & Star Rating Updates"
+date: 2021-11-08 10:00:00 +0000
+---
+
+![](/wiki/shared/news/2021-07-27-performance-points-star-rating-updates/star-rating.jpg)
+
+Thanks to the efforts of various star rating contributors, we are excited to announce that a new series of difficulty calculation changes will be going live shortly. This newspost will go into the what, the who, the when and the how, so read on to find out more!
+
+If you find yourself scratching your head when reading, consider consulting the [performance points](/wiki/Performance_points) and [star rating](/wiki/Beatmapping/Star_rating) wiki articles to gain a greater understanding of the topics.
+
+We will be covering only changes made to the **osu!** game mode in this post.
+
+## osu! changes
+
+As a reminder of the various core values in the difficulty calculation algorithm:
+
+- **Aim**, which considers how difficult it is to consistently hit notes back-to-back during play (affected by things like approach rate, jumps, Flashlight, Hidden, Hard Rock and more)
+- **Speed**, which considers how quickly new elements are presented to the player during the course of the map (affected by things like the number of hit objects over time, BPM, Double Time and Half Time)
+- **Accuracy**, which considers the player's ability to stay on time rhythmically and how difficult this is to achieve (affected by things like Hidden, Hard Rock, Flashlight and more)
+- **Strain**, which considers how often and how long a player has to endure intense or technically demanding patterns during play (affected by things like streams, fast jumps and more)
+
+To help facilitate faster review and uptake of incoming changes, a Performance Points Committee has been formed, consisting of [**StanR**](https://osu.ppy.sh/users/7217455), [**emu1337**](https://osu.ppy.sh/users/2185987), [**MBmasher**](https://osu.ppy.sh/users/4498616) and [**Apo11o**](https://osu.ppy.sh/users/9558549).
```

Comment:
The first public reference I can find about it is [this discord message](https://discord.com/channels/188630481301012481/380598781432823815/883167719942414366).

I believe the rationale for forming it is just to ensure consensus and coherent movement with the regard to how changes are made, to avoid back-and-forths.